lib/overcommit/hook/pre_commit/es_lint.rb

```diff
@@ -8,9 +8,22 @@ module Overcommit::Hook::PreCommit
   # somewhere else. Example:
   #
   #   EsLint:
+  #     required_executable: 'npm'
   #     enabled: true
   #     command: ['npm', 'run', 'lint']
   #
+  # And currently this hook supports only eslint compact format, you can configure
+  # this in package.json. Example:
```

Comment:
You misunderstand; the change @lencioni proposed should be made in your `.overcommit.yml`, not `package.json`.